SIL.Core.Tests/packages.config

```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="NUnit" version="2.6.4" targetFramework="net461" />
-</packages>
```

Comment:
Just curious: how/where does NUnit get referenced now? (Or why/how is it not needed?)

Reply:
For SDK style csproj files they are referenced in the .csproj file.